scss/main.scss

```diff
@@ -110,13 +110,6 @@ hr.hr-gradient {
 
 form {
   font-family: inherit;
-
-  .inline.error {
```

Comment:
moved this to `forms.scss` since this is for overriding rules from `react-formbuilder`